libs/labelbox/tests/data/export/conftest.py

```diff
@@ -118,18 +118,174 @@ def ontology():
         "color": "#008941",
         "classifications": [],
     }
-    entity_tool = {
+    raster_segmentation_tool = {
         "required": False,
-        "name": "entity--",
-        "tool": "named-entity",
-        "color": "#006FA6",
+        "name": "segmentation_mask",
+        "tool": "raster-segmentation",
+        "color": "#ff0000",
         "classifications": [],
     }
-    segmentation_tool = {
+    checklist = {
         "required": False,
-        "name": "segmentation--",
-        "tool": "superpixel",
-        "color": "#A30059",
+        "instructions": "checklist",
+        "name": "checklist",
+        "type": "checklist",
+        "options": [
+            {"label": "option1", "value": "option1"},
+            {"label": "option2", "value": "option2"},
+            {"label": "optionN", "value": "optionn"},
+        ],
+    }
+
+    free_form_text = {
+        "required": False,
+        "instructions": "text",
+        "name": "text",
+        "type": "text",
+        "options": [],
+    }
+
+    radio = {
+        "required": False,
+        "instructions": "radio",
+        "name": "radio",
+        "type": "radio",
+        "options": [
+            {
+                "label": "first_radio_answer",
+                "value": "first_radio_answer",
+                "options": [],
+            },
+            {
+                "label": "second_radio_answer",
+                "value": "second_radio_answer",
+                "options": [],
+            },
+        ],
+    }
+
+    tools = [
+        bbox_tool,
+        bbox_tool_with_nested_text,
+        polygon_tool,
+        polyline_tool,
+        point_tool,
+        raster_segmentation_tool,
+    ]
+    classifications = [
+        checklist,
+        free_form_text,
+        radio,
+    ]
+    ontology = client.create_ontology(
+        "image ontology",
+        {"tools": tools, "classifications": classifications},
+        MediaType.Image,
+    )
+    return ontology
+
+
+@pytest.fixture
+def video_ontology(client: Client):
```

Comment:
I had to make a specific video ontology with the correct features. Since you can not have ontologies anymore with all features

Reply:
We also might move away from not having a media type specified with ontology

Reply:
Do we need a story for that

Reply:
Good point ill write a ticket. But I think we should I was talking to perception and we have had issues revolving around media types not being in ontologies